src/main/java/games/strategy/util/Util.java

```diff
@@ -122,16 +120,6 @@ public static String createUniqueTimeStamp() {
     });
   }
 
-  public static String getStringFromInputStream(InputStream in) {
-    StringBuilder builder = new StringBuilder();
-    try (Scanner scanner = new Scanner(in)) {
-      while (scanner.hasNextLine()) {
-        builder.append(scanner.nextLine()).append("\n");
```

Comment:
Are you sure that the Apache util also appends an end line in the string like this? Just want to make sure the string format is the same.

Reply:
No I'm not, but I believe apache is actually more accurate, so if \r\n is provided as line break \r\n will be used.
Shouldn't change any behaviour though: The old apache httpclient had a "getString" method, which provided the direct string as well, they probably used the same algorithm as in their IOUtils lib, so it worked earlier, it is going to work now!